src/main/java/io/jenkins/stapler/idea/jelly/symbols/SvgFileType.java

```diff
@@ -0,0 +1,33 @@
+package io.jenkins.stapler.idea.jelly.symbols;
+
+import com.intellij.openapi.fileTypes.FileType;
+import org.jetbrains.annotations.NotNull;
+
+public class SvgFileType implements FileType {
```

Comment:
I am really weary of adding file types for files that JetBrains clearly has a file type for already. I messed up peoples IDEs by experimenting with XML file type. This one is not registered, though, but still.

`com.intellij.platform.images` plugin has the type for the SVG.
https://github.com/JetBrains/intellij-community/blob/idea/243.22562.145/images/resources/META-INF/plugin.xml#L29-L30

Reply:
Thanks for finding that! After a bit of fiddling its now working :)